tests/test_systrace.py

```diff
@@ -64,6 +64,14 @@ def test_systrace_userspace(self):
         self.assertTrue(dfr['func'].iloc[2], 'notifyFramePending')
         self.assertTrue(dfr['data'].iloc[-2], 'HW_VSYNC_0')
 
+    def test_systrace_line_num(self):
+        """Test for line numbers in a systrace"""
+        trace = trappy.SysTrace("trace_sf.html")
+        dfr = trace.sched_switch.data_frame
+        self.assertEquals(dfr['__line'].iloc[0], 0)
+        self.assertEquals(dfr['__line'].iloc[1], 6)
+        self.assertEquals(dfr['__line'].iloc[-1], trace.lines - 1)
```

Comment:
Perhaps it's more strict to check for an hardcoded number.

Indeed the values for `__line` are set out of the value of `trace.lines` and since the `sched_switch` is the last event in the test trace I would expect it always matches. However, this is not checking if we should screw-up on updating `trace.lines`, isn't it?